### PR TITLE
adds activated, formatted price and id to product display in admin interface

### PR DESCRIPTION
**Pull request by VirtualSatai**

### CODE REVIEW

File: stregsystem/models.py [hunk 1]
```diff
@@ -198,10 +198,10 @@ class Product(models.Model): # id automatisk...
     deactivate_date = models.DateTimeField(blank=True, null=True)
 
     def activated(self):
```
Comment (DelusionalLogic) [inline stregsystem/models.py]: Maybe this should be in the admin class

File: stregsystem/admin.py [hunk 2]
```diff
@@ -43,9 +43,16 @@ def toggle_active_selected_products(modeladmin, request, queryset):
 class ProductAdmin(admin.ModelAdmin):
     search_fields = ('name', 'price', 'id')
     list_filter = ('deactivate_date', 'price')
-    list_display = ('name', 'price')
+    list_display = ('activated', 'id', 'name', 'get_price_display',)
```
Comment (DelusionalLogic) [inline stregsystem/admin.py]: Is this order correct?

Comment (VirtualSatai) [inline stregsystem/admin.py]: Yes

Comment (DelusionalLogic) [inline stregsystem/admin.py]: Let's ask someone

**Pull request merged**